src/smartAuthorizationHelper.ts

```diff
@@ -129,6 +142,7 @@ export function hasSystemAccess(usableScopes: string[], resourceType: string): b
 export function hasAccessToResource(
     fhirUserObject: FhirResource,
     patientLaunchContext: FhirResource,
+    patientOrgsClaim: FhirResource,
```

Comment:
Let's rename this patientOrgs instead. The claim part isn't helpful in this context.